engine/combat/appraisals/valere/crescent_arc.py

```diff
@@ -13,13 +13,16 @@
 
 
 class CrescentArc(SoSAppraisal):
-    def __init__(self: Self) -> None:
+    def __init__(
+        self: Self, value: int = 0, timing_type: SoSTimingType = SoSTimingType.OneHit
+    ) -> None:
         super().__init__()
-        self.timing_type = SoSTimingType.OneHit
+        self.value = value
+        self.timing_type = timing_type
         self.battle_command = SoSBattleCommand.Skill
         self.target_type = SoSTargetType.Enemy
         # this needs to move to a system that tracks available abilities.
         # May take significant work to determine this unless we do it manually.
         self.skill_command_index = 0
         self.resource = SoSResource.Mana
-        self.cost = 7
+        self.cost = 6
```

Comment:
Just as a note, there is gear that reduces the skill cost. Not sure it's good to get it route wise, but we might need to account for that somewhere.